test/Benchmarks/Metrics/MetricsBenchmarks.cs

```diff
@@ -30,26 +30,27 @@
 // * Summary *
 
 BenchmarkDotNet=v0.13.1, OS=Windows 10.0.19043.1288 (21H1/May2021Update)
```

Comment:
My machine's net5.0 version summary:

BenchmarkDotNet=v0.13.1, OS=Windows 10.0.19043.1288 (21H1/May2021Update)
Intel Xeon CPU E5-1650 v4 3.60GHz, 1 CPU, 12 logical and 6 physical cores
.NET SDK=6.0.100
  [Host]     : .NET 5.0.12 (5.0.1221.52207), X64 RyuJIT  [AttachedDebugger]
  DefaultJob : .NET 5.0.12 (5.0.1221.52207), X64 RyuJIT


|                    Method | AggregationTemporality |        Mean |     Error |    StdDev | Allocated |
|-------------------------- |----------------------- |------------:|----------:|----------:|----------:|
|            CounterHotPath |             Cumulative |    21.13 ns |  0.450 ns |  0.800 ns |         - |
| CounterWith1LabelsHotPath |             Cumulative |   110.44 ns |  2.232 ns |  3.409 ns |         - |
| CounterWith3LabelsHotPath |             Cumulative |   508.15 ns | 10.043 ns | 17.852 ns |         - |
| CounterWith5LabelsHotPath |             Cumulative |   786.85 ns | 15.701 ns | 18.082 ns |         - |
| CounterWith6LabelsHotPath |             Cumulative |   911.47 ns | 16.663 ns | 24.425 ns |         - |
| CounterWith7LabelsHotPath |             Cumulative | 1,052.50 ns | 20.917 ns | 45.026 ns |         - |
|            CounterHotPath |                  Delta |    20.88 ns |  0.447 ns |  0.696 ns |         - |
| CounterWith1LabelsHotPath |                  Delta |   107.41 ns |  2.121 ns |  2.683 ns |         - |
| CounterWith3LabelsHotPath |                  Delta |   507.08 ns | 10.076 ns | 16.271 ns |         - |
| CounterWith5LabelsHotPath |                  Delta |   798.88 ns | 15.786 ns | 34.317 ns |         - |
| CounterWith6LabelsHotPath |                  Delta |   938.21 ns | 18.488 ns | 32.379 ns |         - |
| CounterWith7LabelsHotPath |                  Delta | 1,052.48 ns | 20.964 ns | 29.389 ns |         - |